src/torchmetrics/__init__.py

```diff
@@ -14,6 +14,9 @@
 _PACKAGE_ROOT = os.path.dirname(__file__)
 _PROJECT_ROOT = os.path.dirname(_PACKAGE_ROOT)
 
+if package_available("pretty_errors"):
+    import pretty_errors  # noqa: F401
```

Comment:
It seems like importing torchmetrics now has the effect of changing error messages for all other code. This seems undesirable. Couldn't this be left up to the user? Or the dependency made optional?

Reply:
Also seems unmaintained: https://github.com/onelivesleft/PrettyErrors/issues/57

Reply:
sure, check #2527